in_toto/model.go

```diff
@@ -985,10 +987,16 @@ type Statement struct {
 	Predicate interface{} `json:"predicate"`
 }
 
-// ProvenanceStatement is the definition for an entire provenance statement.
-type ProvenanceStatement struct {
```

Comment:
Should we keep this around and deprecate it so that users switch to explicitly versioned types?

Reply:
SG! Changed!

Reply:
SG! Should I add this comment after slsa v1 comes out or now?

Reply:
Adding `// Deprecated: ..` sounds good to me